app/models.py

```diff
@@ -339,3 +340,64 @@ class UserSkill(db.Model): #pylint: disable=no-init,too-few-public-methods
     user_id = Column(types.Integer, ForeignKey('users.id'), nullable=False)
 
     __table_args__ = (UniqueConstraint('user_id', 'name'),)
+
+class Event(db.Model):
+    '''
+    An event that shows up in the activity feed for a deployment.
+    '''
+
+    __tablename__ = 'events'
+    id = Column(types.Integer, autoincrement=True, primary_key=True)  #pylint: disable=invalid-name
+    created_at = Column(types.DateTime(), default=datetime.datetime.now)
+    updated_at = Column(types.DateTime(), default=datetime.datetime.now,
+                        onupdate=datetime.datetime.now)
+    type = Column(types.String)
+
+    deployment = Column(types.String, nullable=False,
+                        default=get_current_deployment)
+
+    __mapper_args__ = {
+        'polymorphic_identity': 'event',
+        'polymorphic_on': type
+    }
+
+    @classmethod
+    def query_in_deployment(cls):
+        '''
+        Query for events within this deployment
+        '''
+        return cls.query.filter(cls.deployment.in_(current_app.config['SEARCH_DEPLOYMENTS']))
```

Comment:
Since the `deployment` field and `query_in_deployment` are both identical to that of the `User` model, it might be useful to create a `DeploymentMixin` that defines them, to keep our code DRY.